pygpsclient/settings_frame.py

```diff
@@ -453,32 +474,36 @@ def _do_layout(self):
         self._chk_webmap.grid(column=0, row=5, padx=2, pady=2, sticky=W)
         self._lbl_mapzoom.grid(column=1, row=5, sticky=E)
         self._scl_mapzoom.grid(column=2, row=5, columnspan=2, sticky=W)
+        self._lbl_scatterscl.grid(column=1, row=6, sticky=E)

```

Comment:
as per previous comment - would much prefer any scale slider to be placed on the widget itself.